src/core/core.scale.js

```diff
@@ -593,8 +601,13 @@ var Scale = Element.extend({
 
 		me.handleMargins();
 
-		me.width = minSize.width;
-		me.height = minSize.height;
+		if (isHorizontal) {
+			me.width = me._length = chart.width - me.margins.left - me.margins.right;
```

Comment:
`me._length` is initialized in `_configure`, why does it need to be changed here again with a different formula?

Reply:
`me._length` needs to be kept in sync with `me.width` or `me.height` as appropriate depending on orientation